indico/htdocs/sass/partials/_inputs.scss

```diff
@@ -830,3 +830,44 @@ dd > .i-has-action {
         padding: 0.25em 0;
     }
 }
+
+
+.color-palette {
```

Comment:
I would leave `_inputs.scss` for HTML inputs and move `.color-palette` to a dedicated file.